std/src/main/scala/algebra/std/set.scala

```diff
@@ -6,9 +6,8 @@ import algebra.ring.{BoolRng, Semiring}
 
 package object set extends SetInstances
 
-trait SetInstances {
+trait SetInstances extends cats.kernel.std.SetInstances {
   implicit def setLattice[A]: GenBool[Set[A]] = new SetLattice[A]
-  implicit def setPartialOrder[A]: PartialOrder[Set[A]] = new SetPartialOrder[A]
   implicit def setSemiring[A] = new SetSemiring[A]
   def setBoolRng[A] = new SetBoolRng[A]
```

Comment:
And I'm happy to add these comments after this PR is merged, so as not to ask @non to fix deficiencies introduced by me.